tests/test_apply.py

```diff
@@ -418,19 +424,19 @@ class TestExpval:
         [
             (qml.PauliX, [1 / math.sqrt(2), 1 / math.sqrt(2)], 1),
             (qml.PauliX, [1 / math.sqrt(2), -1 / math.sqrt(2)], -1),
-            (qml.PauliX, [1, 0], 0),
-            (qml.PauliY, [1 / math.sqrt(2), 1j / math.sqrt(2)], 1),
-            (qml.PauliY, [1 / math.sqrt(2), -1j / math.sqrt(2)], -1),
-            (qml.PauliY, [1, 0], 0),
-            (qml.PauliZ, [1, 0], 1),
-            (qml.PauliZ, [0, 1], -1),
-            (qml.PauliZ, [1 / math.sqrt(2), 1 / math.sqrt(2)], 0),
-            (qml.Hadamard, [1, 0], 1 / math.sqrt(2)),
-            (qml.Hadamard, [0, 1], -1 / math.sqrt(2)),
-            (qml.Hadamard, [1 / math.sqrt(2), 1 / math.sqrt(2)], 1 / math.sqrt(2)),
-            (qml.Identity, [1, 0], 1),
-            (qml.Identity, [0, 1], 1),
-            (qml.Identity, [1 / math.sqrt(2), -1 / math.sqrt(2)], 1),
+            # (qml.PauliX, [1, 0], 0),
+            # (qml.PauliY, [1 / math.sqrt(2), 1j / math.sqrt(2)], 1),
+            # (qml.PauliY, [1 / math.sqrt(2), -1j / math.sqrt(2)], -1),
+            # (qml.PauliY, [1, 0], 0),
+            # (qml.PauliZ, [1, 0], 1),
+            # (qml.PauliZ, [0, 1], -1),
+            # (qml.PauliZ, [1 / math.sqrt(2), 1 / math.sqrt(2)], 0),
+            # (qml.Hadamard, [1, 0], 1 / math.sqrt(2)),
+            # (qml.Hadamard, [0, 1], -1 / math.sqrt(2)),
+            # (qml.Hadamard, [1 / math.sqrt(2), 1 / math.sqrt(2)], 1 / math.sqrt(2)),
+            # (qml.Identity, [1, 0], 1),
+            # (qml.Identity, [0, 1], 1),
+            # (qml.Identity, [1 / math.sqrt(2), -1 / math.sqrt(2)], 1),
```

Comment:
Are these failing?

Reply:
Good catch! No. I just forgot to uncomment them after debugging.